packages/common/vbmStatus.ts

```diff
@@ -27,17 +28,10 @@ export const vbmStatus = (state: State): Status => {
     return { status: "VbmApp" }
   }
 
-  switch (state) {
-    case "Pennsylvania": return {
-      status: "Website",
-      regUrl: "https://www.pavoterservices.pa.gov/OnlineAbsenteeApplication/#/OnlineAbsenteeBegin",
-      infoUrl: "https://www.votespa.com/Voting-in-PA/Pages/Mail-and-Absentee-Ballot.aspx",
-    }
-    case "Hawaii": return { status: "Automatic" }
-    case "Colorado": return { status: "Automatic" }
-    case "Utah": return { status: "Automatic" }
-    case "Oregon": return { status: "Automatic" }
-    case "Washington": return { status: "Automatic" }
+  const electionPortal = getStatePortal(state)
```

Comment:
We want to keep this code.  It's for online VBM signup in states that we don't support.